lms/static/scripts/ui-playground/components/ToolbarPage.tsx

```diff
@@ -0,0 +1,283 @@
+        title="Toolbar UI"
+        {...modalProps}
+        classes={classnames('w-[90vw]', classes)}
+        width="custom"
+        onClose={closeModal}
+      >
+        <div className="border">{children}</div>
+      </Modal>
+    );
+  }
+}
+
+export default function ToolbarPage() {
+  return (
+    <Library.Page
+      title="Instructor Toolbar"
+      intro={
+        <p>
+          These sketches explore some UI possibilities for the Instructor
+          Toolbar: adding an edit-assignment link and potentially having a
+          collapsible view.
+        </p>
+      }
+    >
+      <Library.Section
+        title="Full Toolbar layout sketches"
+        intro={
+          <p>
+            Layout examples are shown in modals to provide more available screen
+            width. You can resize your browser to see how the layouts adapt to
+            different viewports.
+          </p>
+        }
+      >
+        <Library.Pattern title="Starting point UI">
+          <Library.Demo title="Starting-point UI">
+            <ToolbarExample title="Starting-point UI">
+              <>
+                <header
+                  className={classnames(
+                    'grid grid-cols-1 items-center gap-y-2 p-2',
+                    'lg:grid-cols-3 lg:gap-x-4 lg:px-3'
+                  )}
+                >
+                  <div className="space-y-1">
+                    <h1
+                      className="text-lg font-semibold leading-none"
+                      data-testid="assignment-name"
+                    >
+                      How four bananas changed the world
+                    </h1>
+                    <h2
+                      className="text-sm font-normal text-color-text-light leading-none"
+                      data-testid="course-name"
+                    >
+                      Fruit and its incredible power
+                    </h2>
+                  </div>
+
+                  <div
+                    className={classnames(
+                      'flex flex-col gap-2',
+                      'sm:flex-row',
+                      'lg:col-span-2 lg:gap-4 ' /* cols 2-3 of 3 */
+                    )}
+                  >
+                    <div className="flex-grow-0 sm:flex-grow">
+                      <div
+                        className={classnames(
+                          // Narrower widths: label above field
+                          'flex flex-col gap-1',
+                          // Wider widths: label to left of field
+                          'xl:flex-row xl:gap-3 xl:items-center'
+                        )}
+                      >
+                        <label
+                          className={classnames(
+                            'flex-grow font-medium text-sm leading-none',
+                            'xl:text-right'
+                          )}
+                          data-testid="student-selector-label"
+                          htmlFor="student-selector"
```

Comment:
It doesn't really matter that much for a UI prototype page, but we should get in the habit of using `useId` or our wrapper around that hook for generating IDs in components.